src/testRunners/jasmine/jasmine2.js

```diff
@@ -65,9 +67,20 @@ function jasmine2(config, environment, moduleLoader, testPath) {
               throw Error('lastCalledWith() should be used on a mock function');
             }
             const calls = actual.mock.calls;
-            const args = Array.prototype.slice.call(arguments, 1);
+            const expected = Array.prototype.slice.call(arguments, 1);
+            const actualValues = calls[calls.length - 1];
+            const pass = util.equals(actualValues, expected);
+            const message = [
+              'function wasn\'t called with the expected values.',
+              'Expected:',
+              JSON.stringify(expected),
```

Comment:
jasmine2 unfortunately always creates the message instead of lazily creating it (jasmine 1 had a `message()` callback). I'm really worried that stringifying large objects will incur a non-trivial perf slowdown on www. Can we instead just print `expected` (or spread their values into the message) here? That shouldn't be worse for perf and the message should be good enough.